### ISSUE TITLE
AttributeField's equals method throws NullPointerException

### ISSUE THREAD

**Issue by dinoopp** [usnistgov/jsip#25]
gov.nist.javax.sdp.fields.AttributeField.equals(Object that) throws NullPointerException if attributes having no name even though both are same. 

Eg: Comparing two a=sendrecv  attributes causes NullPointerException.

Suggestion, if both the attributes doesn't have a name , then  no need of comparing it

**Comment by vladimirralev** [comment]
You are right. Root cause is the attrib value here. I will take care of it. Thanks.
